Formula/node.rb

```diff
@@ -31,7 +31,7 @@ class Node < Formula
   uses_from_macos "zlib"
 
   on_linux do
-    depends_on "gcc" => :build
+    depends_on "gcc"
```

Comment:
How about this?
```suggestion
    depends_on "gcc@10" => :build
```
This makes the bottle work without installing Homebrew `gcc` on Ubuntu 20.04, but I don't know about other distros.

Reply:
Nope. It's either gcc or nothing currently.